management-api-agent-3.x/src/main/java/com/datastax/mgmtapi/rpc/ObjectSerializer3x.java

```diff
@@ -73,7 +73,7 @@ public ObjectSerializer3x(Class<T> clazz, Type genericType) {
                             field -> field.getName(),
                             field ->
                                 new FieldSerializer(
-                                    GenericSerializer3x.getType(field.getType()), field))));
+                                    GenericSerializer3x.getType(field.getGenericType()), field))));
```

Comment:
Issue:
We probably need this in the DSE 7 ObjectSerializer class as well. Unfortunately with the dependencies being internal Cassandra/DSE classes, we have to duplicate a lot of this to compile against the correct versions.

Reply:
👍 I missed it because the dse7 profile was not enabled in my IDE.